packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -150,15 +148,15 @@ class ConnectionStateHandlerPassThrough
 	public initProtocol(protocol: IProtocolHandler) {
 		return this.pimpl.initProtocol(protocol);
 	}
-	public receivedDisconnectEvent(reason: string, error?: IAnyDriverError) {
-		return this.pimpl.receivedDisconnectEvent(reason, error);
+	public receivedDisconnectEvent(reason: IConnectionStateChangeReason) {
```

Comment:
Should this also be `IConnectionStateChangeReason<IAnyDriverError>`?

Reply:
```suggestion
	public receivedDisconnectEvent(reason: IConnectionStateChangeReason<IAnyDriverError>) {
```